### PR TITLE
Parent POM update

### PR DESCRIPTION
**Pull request by oleg-nenashev**
- [x] Pick the latests Parent POM
- [x] Escape Jellys by default
- [x] Fix FindBugs
- [x] Not FindBugs: Synchrnize access to the MaskPasswordConfig's maskPasswordsParamValues cache

It is required for the next release.

@reviewbybees

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
```
[INFO] Exception is caught when Exception is not thrown in new com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsConfig$1(MaskPasswordsConfig, Class) [com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsConfig$1] At MaskPasswordsConfig.java:[line 311] REC_CATCH_EXCEPTION
```

Does not appear on my laptop...

**Comment by aheritier** [comment]
:bee:

File: src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsOutputStream.java [hunk 1]
```diff
@@ -119,11 +120,11 @@ public MaskPasswordsOutputStream(OutputStream logger, @CheckForNull Collection<S
 
     @Override
     protected void eol(byte[] bytes, int len) throws IOException {
-        String line = new String(bytes, 0, len);
+        String line = new String(bytes, 0, len, "UTF-8");
```
Comment (jglick) [inline src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsOutputStream.java]: This is probably incorrect; in general you would need to pay attention to the `Run.charset`. However the code was already wrong, as it used the default encoding on master.

File: src/main/resources/com/michelin/cio/hudson/plugins/passwordparam/PasswordParameterDefinition/config.jelly [hunk 2]
```diff
@@ -22,7 +22,7 @@
   - OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
   - THE SOFTWARE.
   -->
-
+<?jelly escape-by-default='true'?>
```
Comment (jglick) [inline src/main/resources/com/michelin/cio/hudson/plugins/passwordparam/PasswordParameterDefinition/config.jelly]: Check behavior with `instance.description`.

File: src/main/resources/com/michelin/cio/hudson/plugins/passwordparam/PasswordParameterDefinition/index.jelly [hunk 3]
```diff
@@ -22,7 +22,7 @@
   - OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
   - THE SOFTWARE.
   -->
-
```
Comment (jglick) [inline src/main/resources/com/michelin/cio/hudson/plugins/passwordparam/PasswordParameterDefinition/index.jelly]: Ditto.

File: src/main/resources/com/michelin/cio/hudson/plugins/passwordparam/PasswordParameterValue/value.jelly [hunk 4]
```diff
@@ -22,7 +22,7 @@
   - OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
   - THE SOFTWARE.
   -->
-
+<?jelly escape-by-default='true'?>
```
Comment (jglick) [inline src/main/resources/com/michelin/cio/hudson/plugins/passwordparam/PasswordParameterValue/value.jelly]: Ditto.

File: src/main/resources/com/michelin/cio/hudson/plugins/passwordparam/PasswordParameterDefinition/config.jelly (hunk 2 above)
Comment (oleg-nenashev) [inline src/main/resources/com/michelin/cio/hudson/plugins/passwordparam/PasswordParameterDefinition/config.jelly]: It is within the text area, hence it won't be formatted AFAIK

File: src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsOutputStream.java (hunk 1 above)
Comment (oleg-nenashev) [inline src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsOutputStream.java]: Do you offer to take the less wrong option?

File: src/main/resources/com/michelin/cio/hudson/plugins/passwordparam/PasswordParameterDefinition/config.jelly (hunk 2 above)
Comment (oleg-nenashev) [inline src/main/resources/com/michelin/cio/hudson/plugins/passwordparam/PasswordParameterDefinition/config.jelly]: Tested manually, works well for me

File: src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsOutputStream.java (hunk 1 above)
Comment (jglick) [inline src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsOutputStream.java]: I suppose the more conservative change would be to restore the original code, and add

```java
@SuppressFBWarning(value="whatever", justification="TODO should probably be using Run.charset")
```

**Comment by oleg-nenashev** [comment]
@reviewbybees done

**Pull request merged**